ml/src/test/scala/frameless/ml/TypedDatasetSuite.scala

```diff
@@ -0,0 +1,27 @@
+package frameless.ml
+
+import org.apache.spark.{SparkConf, SparkContext}
+import org.apache.spark.sql.{SQLContext, SparkSession}
+import org.scalactic.anyvals.PosZInt
+import org.scalatest.FunSuite
+import org.scalatest.prop.Checkers
+
+trait SparkTesting {
```

Comment:
I think we need to generalize this code and don't copy paste it among project. Maybe add this to frameless-core?

Reply:
The issue if I add this the frameless-core is that TypedDatasuite will no longer belong to the "test" scope, is it not?